ietf/status/views.py

```diff
@@ -0,0 +1,50 @@
+# Copyright The IETF Trust 2016-2020, All Rights Reserved
+# -*- coding: utf-8 -*-
+
+import json
+from django.http import HttpResponse, HttpResponseRedirect, HttpResponseNotFound
+from ietf.utils import markdown
+from django.core.exceptions import FieldError
+from django.shortcuts import render, get_object_or_404
+from ietf.status.models import Status
+
+import debug                            # pyflakes:ignore
+
+def get_context_data():
+    status = Status.objects.order_by("-date").first()
+    if status is None or status.active == False:
+        return { "hasMessage": False }
```

Comment:
This prevents us from staging the next message.
Consider instead:
```
    status = Status.objects.filter(active=True).order_by("-date").first()
```

Reply:
Changed. This was intentional (ie only the latest status which must also be active is used) but happy to change it.